Formula/c/ctags.rb

```diff
@@ -91,6 +91,7 @@ def caveats
     EOS
     system bin/"ctags", "-R", "."
     assert_match(/func.*test\.c/, File.read("tags"))
-    assert_match "+regex", shell_output("#{bin}/ctags --version")
```

Comment:
Sounds like we should be investigating why the `regex` feature isn't being built before deciding we no longer want to ship it.

Reply:
make sense